hetio/neo4j.py

```diff
@@ -89,6 +96,10 @@ def append(self, x):
             self.create()
 
     def create(self):
+        import py2neo
```

Comment:
let's do all py2neo imports using `import_py2neo` to gaurantee timeout has been set if necessary.